packages/components/src/mobile-promo-card/index.tsx

```diff
@@ -14,41 +14,32 @@ export type MobilePromoCardProps = {
 };
 
 export default function MobilePromoCard( { className, isWoo }: MobilePromoCardProps ) {
-	// Using useTranslate() with interpolation to set up the title/message.
-	// https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md
 	const translate = useTranslate();
-	const redirectLink = isWoo ? 'https://woo.com/mobile/' : 'https://jetpack.com/app/';
-	const linkClassName = isWoo ? 'woo' : 'jetpack';
-	const components = {
-		a: <a className={ linkClassName } href={ redirectLink } />,
-	};
 	// Basic user agent testing so we can show app store badges on moble.
 	const userAgent = window.navigator.userAgent.toLowerCase();
 	const isApple = userAgent.includes( 'iphone' ) || userAgent.includes( 'ipad' );
 	const isGoogle = userAgent.includes( 'android' );
 
-	const getTitle = () => {
-		if ( isWoo ) {
-			return translate( 'Bring your Store stats with you using the {{a}}Woo{{/a}} mobile app', {
-				components,
-			} );
-		}
-		return translate( 'Bring your stats with you using the {{a}}Jetpack{{/a}} mobile app', {
-			components,
-		} );
-	};
-
 	// Determines message text based on mobile, tablet, or Desktop.
 	const getMessage = () => {
 		if ( isApple ) {
 			return translate(
 				'Check your stats on-the-go and get real-time notifications with the Jetpack mobile app.'
 			);
-		} else if ( isGoogle ) {
+		}
+		if ( isGoogle ) {
```

Comment:
Should we check the `isWoo` within `isGoogle` to replace the mobile app name?

Reply:
This is actually handled in a follow-up PR. I was intentionally keeping this one small to make sure the build tests would pass as they were failing in my larger PR.

Reply:
That sounds like a plan.